x-pack/plugins/security_solution/public/common/components/search_bar/index.tsx

```diff
@@ -351,15 +351,27 @@ export const dispatchUpdateSearch = (dispatch: Dispatch) => ({
     const fromDate = formatDate(start);
     let toDate = formatDate(end, { roundUp: true });
     if (isQuickSelection) {
-      dispatch(
-        inputsActions.setRelativeRangeDatePicker({
-          id,
-          fromStr: start,
-          toStr: end,
-          from: fromDate,
-          to: toDate,
-        })
-      );
+      if (end === start) {
+        dispatch(
+          inputsActions.setAbsoluteRangeDatePicker({
+            id,
+            fromStr: start,
+            toStr: end,
+            from: fromDate,
+            to: toDate,
```

Comment:
Seeing an issue where the date picker enters an invalid state:

1. Open a new browser tab, and navigate to the Kibana home page i.e. http://localhost:5601/app/home#/
2. Click the `Security` button to navigate to the Security solution `Overview` page
3. Click the Timeline flyout button to open Timeline
4. Type the following KQL query: `user.name: *`, then press `Enter`

**Expected results**
- Data is displayed in Timeline
- Timeline's date picker still displays `Last 24 hours`
- The `Overview` page still displays data

5. Click Timeline's `Refresh` button

**Expected results**
- The data displayed in Timeline is refreshed
- Timeline's date picker still displays `Last 24 hours`
- The `Overview` page refreshes (because the synchronized date has been updated) and displays data

6. Change Timeline's date picker from `Last 24 hours` to `Today`

**Expected result**
- The data displayed in Timeline is updated to reflect the new date range
- Timeline's date picker displays `Today`
- The `Overview` page refreshes (because the synchronized date has been updated) and displays data

7. Once again, click Timeline's `Refresh` button (this time with `Today` selected)

**Expected results**
- The data displayed in Timeline is updated (to show any new data that may have arrived)
- Timeline's date picker displays `Today`
- The `Overview` page refreshes (because the synchronized date has been updated) and displays data

**Actual results**
- Data is **not** displayed in Timeline (`0 of 0 events`)
- Timeline's date picker changes to an invalid absolute date range, where both the `Start date` and `End date` are the same, e.g. `Nov 3, 2020 @ 00:00:00.000`
- The `Overview` page refreshes (because the synchronized date has been updated) and, after some time, displays one or more toast errors (i.e. `Failed to query events data`, `Failed to query alerts data`), per the following details / screenshot

```
illegal_argument_exception
Zero or negative time interval not supported

Error: EsError
    at EnhancedSearchInterceptor.handleSearchError (http://localhost:5601/9007199254740991/bundles/plugin/data/data.plugin.js:46110:16)
    at CatchSubscriber.selector (http://localhost:5601/9007199254740991/bundles/plugin/dataEnhanced/dataEnhanced.plugin.js:1049:75)
    at CatchSubscriber.__kbnSharedDeps__.../../node_modules/rxjs/_esm5/internal/operators/catchError.js.CatchSubscriber.error (http://localhost:5601/9007199254740991/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:350686:31)
    at TakeUntilSubscriber.__kbnSharedDeps__.../../node_modules/rxjs/_esm5/internal/Subscriber.js.Subscriber._error (http://localhost:5601/9007199254740991/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:347541:26)
    at TakeUntilSubscriber.__kbnSharedDeps__.../../node_modules/rxjs/_esm5/internal/Subscriber.js.Subscriber.error (http://localhost:5601/9007199254740991/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:347521:18)
    at MergeMapSubscriber.__kbnSharedDeps__.../../node_modules/rxjs/_esm5/internal/Subscriber.js.Subscriber._error (http://localhost:5601/9007199254740991/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:347541:26)
    at MergeMapSubscriber.__kbnSharedDeps__.../../node_modules/rxjs/_esm5/internal/Subscriber.js.Subscriber.error (http://localhost:5601/9007199254740991/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:347521:18)
    at TapSubscriber.__kbnSharedDeps__.../../node_modules/rxjs/_esm5/internal/operators/tap.js.TapSubscriber._error (http://localhost:5601/9007199254740991/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:355364:26)
    at TapSubscriber.__kbnSharedDeps__.../../node_modules/rxjs/_esm5/internal/Subscriber.js.Subscriber.error (http://localhost:5601/9007199254740991/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:347521:18)
    at TakeWhileSubscriber.__kbnSharedDeps__.../../node_modules/rxjs/_esm5/internal/Subscriber.js.Subscriber._error (http://localhost:5601/9007199254740991/bundles/kbn-ui-shared-deps/kbn-ui-shared-deps.js:347541:26)
```

<img width="1699" alt="failed-to-query-alerts-data" src="https://user-images.githubusercontent.com/4459398/98057041-8f6b3a80-1dfe-11eb-86aa-9d4e6c78af3f.png">